src/spinner/styles.scss

```diff
@@ -6,36 +6,44 @@
 @use '../internal/styles' as styles;
 @use './mixins' as mixins;
 @use '../internal/styles/tokens' as awsui;
+@use '../internal/generated/custom-css-properties/index' as custom-props;
```

Comment:
Do we really need to go this way? Why is this not `transform: scaleX(-1);` trick, same as icons?

Reply:
I was hung up on this for a long time. I started with the same solution as icons but for reasons I don't understand I couldn't get a transform to create an identical inversion. Ultimately I had to change the rotation targets to get it to work. I'll DM you a video.